src/extension/background-script/connectors/nwc.ts

```diff
@@ -100,22 +100,27 @@ class NWCConnector implements Connector {
 
   async makeInvoice(args: MakeInvoiceArgs): Promise<MakeInvoiceResponse> {
     const invoice = await this.nwc.makeInvoice({
-      amount: args.amount,
-      defaultMemo: args.memo,
+      amount:
+        typeof args.amount === "number"
+          ? args.amount
+          : parseFloat(args.amount) || 0,
+      description: args.memo,
     });
-
-    const decodedInvoice = lightningPayReq.decode(invoice.paymentRequest);
-    const paymentHash = decodedInvoice.tags.find(
-      (tag) => tag.tagName === "payment_hash"
-    )?.data as string | undefined;
-    if (!paymentHash) {
-      throw new Error("Could not find payment hash in invoice");
+    let rHash = invoice.payment_hash;
+
+    if (!rHash) {
```

Comment:
this is not needed. NWC make_invoice response returns the payment hash and invoice.